gatorgrouper/utils/run.py

```diff
@@ -9,6 +9,7 @@
 from gatorgrouper.utils import display
 from gatorgrouper.utils import constants
 
+# pylint: disable=bad-continuation
```

Comment:
This should be positioned where it is needed -- here it disables this check for the entire file!

Reply:
The reason I put it here is due to that `pylint` warms bad-continuation from line 45 to 52. I don't want to put `pylint disable` for 8 lines.

Reply:
Then put it above the method that those lines are in, that way it is only that method where it is ignored.

Reply:
resolved

Reply:
@Lancasterwu can you also quickly remove those trailing `pylint: disable`s and move them above their lines? Using trailing comments does not make for clean code.

Reply:
Sure

Reply:
fixed